components/Footer/Footer.tsx

```diff
@@ -0,0 +1,64 @@
+import Link from "next/link";
+import React from "react";
+import { Logo } from "../Logo/Logo";
+
+export interface Link {
+  label: string;
+  route: string;
+}
+
+interface FooterLinkProps extends React.HTMLProps<HTMLLIElement> {
+  route: string;
+}
+
+interface FooterProps extends Partial<React.HTMLProps<HTMLElement>> {
+  links: Link[];
+}
+
+const focusClasses = "focus:shadow-md focus:border-white focus:border-[2px]";
+
+export const Footer = ({ links }: FooterProps) => {
+  return (
+    <footer
+      className="flex items-center justify-center bg-blue pb-14 pt-10 md:pt-[52px]"
+      aria-label="Footer"
+      data-testid="footer"
+    >
+      <div className="mx-auto flex w-full max-w-[1040px] flex-col items-center justify-between md:flex-row">
+        <Link
+          className={`border-2 border-transparent outline-none md:pl-2 ${focusClasses}`}
```

Comment:
Maybe for the Logo, we could add 4px of padding. Btw, this isn't specified in Figma.
@TonyMckes what do u think about this?

Reply:
Also before its changes the layout due its width, the active border goes off the element.

![footerJug](https://user-images.githubusercontent.com/96669064/231315641-388db4e8-76f6-446c-8d64-78b04a2f803f.jpg)

Reply:
The same happens when the `Footer` takes the full width of the screen.
Maybe we could replace the left padding with left margin,  and add the padding prop to add free space between the icon and its border.

Reply:
What do you think about adding the rounded border to the logo too? @TonyMckes @alexxuss1996

Reply:
Yeah, all the links need some padding since the text seems too close to the white border.

Hmm... Actually, it needs more than that. There's an existing `Link` component already styled, maybe you can copy some of the styles from there.